source/upgrade/important-upgrade-notes.rst

```diff
@@ -53,6 +53,10 @@ We recommend reviewing the `additional upgrade notes <#additional-upgrade-notes>
 |                                                    |                                                                                                                                                                  |
 |                                                    | MySQL: https://github.com/mattermost/mattermost/blob/master/server/channels/db/migrations/mysql/000139_remoteclusters_add_last_global_user_sync_at.down.sql      |
 |                                                    | and https://github.com/mattermost/mattermost/blob/master/server/channels/db/migrations/mysql/000139_remoteclusters_add_last_global_user_sync_at.up.sql           |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
A few comments, happy to help with these updates as needed:

1. The table is currently broken.
2. The same note could be added to the [changelog](https://docs.mattermost.com/about/mattermost-v10-changelog.html#release-v10-10-feature-release).

Reply:
1. should be fixed and also added 2.